docs/src/developer/reference/oauth.md

```diff
@@ -0,0 +1,402 @@
+# OAuth
+
+OAuth 2.0 is used to authorize 3rd party applications to access `wire-server` resources on behalf of a Wire user.
+
+Currently, only 3rd party apps that have been implemented and approved by Wire are supported, e.g. Outlook Calendar Extension. OAuth is not open for public use.
```

Comment:
where can i get an exhaustive list of supported apps?  (brig settings, correct?  probably something for later down in the doc, but still.)

Reply:
This info is not in brig settings as it is dynamic.
Currently, there is only Outlook calendar integration. I'll list it here.

Reply:
what does "dynamic" mean?  that we only know this as opaque strings?

Reply:
Dynamic means that an OAuth client can be created via the internal API and not via a static config file.